config/settings.rb

```diff
@@ -12,12 +12,12 @@
 SETTINGS.deep_merge!(Foreman::EnvSettingsLoader.new.to_h)
 
 # Force setting to true until all code using it is removed
-[:locations_enabled, :organizations_enabled].each do |setting|
+[:locations_enabled, :organizations_enabled, :unattended].each do |setting|
   SETTINGS[setting] = true
```

Comment:
I wonder if we could implement a special Hash type in development mode that generates a deprecation warning when these settings are used.

Reply:
Probably not worth it for the once in a blue moon when we drop one of these, and by forcing them to true we avoid breaking any code that might still check them